packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -1233,33 +1233,29 @@ function VM-Update-Registry-Value {
     param(
         [Parameter(Mandatory=$true)]
         [ValidateNotNullOrEmpty()]
-        [string] $name,
+        [string] $path,
 
         [Parameter(Mandatory=$true)]
         [ValidateNotNullOrEmpty()]
-        [string] $path,
+        [string] $name,
 
         [Parameter(Mandatory=$true)]
         [ValidateNotNullOrEmpty()]
         [string] $value,
 
         [Parameter(Mandatory=$true)]
         [ValidateSet("String", "ExpandString", "Binary", "DWord", "QWord", "MultiString", "Unknown")]
-        [string] $type,
-
-        [Parameter(Mandatory=$true)]
-        [ValidateNotNullOrEmpty()]
-        [string] $data
```

Comment:
Ah, I missed that one. I was looking at packages that used the function and didn't see it used, so I assumed it was just a mistake. :)

Just seems odd to me to use `$value` for `-Name` in `Set-ItemProperty` and not `$name`, and then use `$data` / `$validatedData` for `-Value` instead of using `$value`.

I would think that changing the naming convention to what `Set-ItemProperty` expects (to mean, also updating the places that use this function) might be a better route to take, but we can address that in another PR. :)